### ISSUE TITLE
[Windows] EncryptedLocalStore doesn't work if user name has chinese symbols

### ISSUE THREAD

**Issue by itlancer** [Gamua/Adobe-Runtime-Support#165]
## Problem Description
EncryptedLocalStore doesn't work if Windows user name has chinese symbols (at least one). For example: 地铁
It just throw exception when you try to use it.
Tested with AIR 32.0.0.89 and AIR 33.0.2.330 with different Windows 10 1903 x64 devices with 32-bit and 64-bit AIR applications with captive runtime .

It works fine for the same devices with the same OS version with other users which doesn't contains chinese symbols.
Alose it works fine with other user names which contains latin, cyrillic symbols, spaces, special characters, numbers but not chinese symbols.
`EncryptedLocalStore.isSupported` always returns `true` in all these cases.

## Steps to Reproduce
1) Create new Windows user with chinese symbols in its name. For example: 地铁
2) Login with new Windows user.
3) Launch code below.

Application example with sources and Scout log attached.
[encrypted_local_store_username_symbols_bug.zip](https://github.com/Gamua/Adobe-Runtime-Support/files/3923984/encrypted_local_store_username_symbols_bug.zip)

```
var bytes:ByteArray = EncryptedLocalStore.getItem("test");
```

**Actual Result:**
Exception will be throwed:
```
Error: EncryptedLocalStore database access error

          Exception: flash.errors::IOError
flash.filesystem::FileStream/open
runtime::FileUtils$/readByteArrayFromFile
runtime::RunningApplicationSignature/_VerifyRunningAppSig
runtime::RunningApplicationSignature/VerifyRunningAppSig
Runtime$/verifyAppSignature
Runtime$/getVerifiedPublisherIdentifier
flash.data::EncryptedLocalStore$/getItemNative
flash.data::EncryptedLocalStore$/getItem
EncryptedLocalStoreUserNameSymbolsBug

          Exception: flash.errors::IOError
flash.data::EncryptedLocalStore$/processErrorCode
flash.data::EncryptedLocalStore$/getItem
EncryptedLocalStoreUserNameSymbolsBug
```

**Expected Result:**
`EncryptedLocalStore.getItem` returns `ByteArray` with data of specified name without exceptions.

## Known Workarounds
none

**Comment by itlancer** [comment]
Issue still exists with latest AIR 33.1.1.190.

**Comment by itlancer** [comment]
It fixed with latest AIR 33.1.1.217. Thank you!
